f2/server.js

```diff
@@ -61,6 +68,17 @@ const uploadData = (req, res) => {
     console.log('Files', files)
     for (const file of Object.entries(files)) {
       console.log(file)
+      //scan file for virus
+      var readStream = fs.createReadStream(file[1].path)
+      scanner
+        .scanStream(readStream, 10000)
```

Comment:
Is the 10000 some sort of timeout? If so maybe just add a comment here.

Reply:
fixed